pipelined/bevy_pbr2/src/material.rs

```diff
@@ -206,16 +209,27 @@ impl RenderAsset for StandardMaterial {
         if material.unlit {
             flags |= StandardMaterialFlags::UNLIT;
         }
-        if material.normal_map_texture.is_some() {
-            flags |= StandardMaterialFlags::NORMAL_MAP_TEXTURE;
-        }
+        let has_normal_map = material.normal_map_texture.is_some();
+        // NOTE: 0.5 is from the glTF default - do we want this?
```

Comment:
I think following the glTF "0.5 is the default cutoff" is reasonable. If people want a different cutoff then they just have to overwrite the cutoff value. @cart

Reply:
Yup this seems reasonable to me. We can adjust later as needed.